spec/acceptance/motd_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper_acceptance'
+
+describe 'static message from content' do
+  it do
+    pp = <<-EOS
+      class { motd:
+        content => "Hello world!\n",
+      }
+    EOS
+    # Run it twice and test for idempotency
+    apply_manifest(pp, :catch_failures => true)
+    apply_manifest(pp, :catch_changes => true)
+  end
+
+
+  if fact('osfamily') == 'Debian' || fact('osfamily') == 'RedHat'
```

Comment:
Just a discussion topic, but would it be better if this said:

~~~
if fact('osfamily') != 'Windows'
~~~

Reasoning is, if we have an motd module, we should eventually (definitely) expand the support to include other *nix platforms that have `/etc/motd` management, like SuSE, AmazonLinux, etc. And it would be easier for the test to exclude windows vs include all the linux distros.